README.md

```diff
@@ -1,225 +1,123 @@
-# GamestonkTerminal 🚀
+<!-- PROJECT SHIELDS -->
+<!--
+*** I'm using markdown "reference style" links for readability.
+*** Reference links are enclosed in brackets [ ] instead of parentheses ( ).
+*** See the bottom of this document for the declaration of the reference variables
+*** for contributors-url, forks-url, etc. This is an optional, concise syntax you may use.
+*** https://www.markdownguide.org/basic-syntax/#reference-style-links
+-->
+[![Stargazers][stars-shield]][stars-url]
+[![Forks][forks-shield]][forks-url]
+[![Contributors][contributors-shield]][contributors-url]
+[![Issues][issues-shield]][issues-url]
+[![MIT License][license-shield]][license-url]
+
+
+
+<!-- PROJECT LOGO -->
+<br />
+<p align="center">
+  <a href="https://github.com/DidierRLopes/GamestonkTerminal">
+    <img src="images/logo.png" alt="Logo" width="80" height="80">
+  </a>
+
+  <h3 align="center">Gamestonk Terminal 🚀</h3>
```

Comment:
This should probably be an h2 or h1, It looks nicer in the original readme where its bigger.

Reply:
I'll try this!

Reply:
I've tried and I think h3 looks better actually, looks more minimal. But let's wait till we get a proper logotype. I have someone working on it during weekend :)